Jenkinsfile

```diff
@@ -108,9 +108,11 @@ def unitTests(os, nodeVersion, npmVersion, testSuiteBranch) {
 							} finally {
 								// Kill the emulators!
 								if ('android'.equals(os)) {
-									sh returnStatus: true, script: 'adb -e shell am force-stop com.appcelerator.testApp.testing'
-									sh returnStatus: true, script: 'adb -e uninstall com.appcelerator.testApp.testing'
-									killAndroidEmulators()
+									timeout(5) {
```

Comment:
may only want to wrap these two add commands and not the killAndroidEmulators() call, which I think just does killall and shouldn't block. (Also looks like only the uninstall command actually blocks)